Automattic/wp-calypso
# Components: Add PromoCard component

#### Proposed Changes

Adds a new `PromoCard` component for use on the Stats dashboard.

![PromoCard](https://user-images.githubusercontent.com/40267301/200283879-6753fe06-1d62-40bd-b2fe-4e44e69b495a.png)

#### Testing Instructions



* Check out this branch locally.
* Spin up Storybook via `yarn workspace @automattic/components run storybook`.
* Check the PromoCard stories. Ensure they behave as expected.

#### Pre-merge Checklist






Related to #69230
